docs/ref/contrib/sites.txt

```diff
@@ -216,6 +216,27 @@ On Lawrence.com, this email has the subject line "Thanks for subscribing to
 lawrence.com alerts." On LJWorld.com, the email has the subject "Thanks for
 subscribing to LJWorld.com alerts." Same goes for the email's message body.
 
+Using ``site`` in templates
+---------------------------
+
+.. versionadded:: 6.0
+
+   Add the :func:`~django.template.context_processors.csp` context processor
+   to your :setting:`TEMPLATES` setting. This makes the current Site object
+   available in the Django templates as the ``site`` context variable::
+
+    TEMPLATES = [
+        {
+            "BACKEND": "django.template.backends.django.DjangoTemplates",
+            "OPTIONS": {
+                "context_processors": [
+                    # ...
+                    "django.contrib.sites.context_processors.site",
+                ],
+            },
+        },
+    ]
```

Comment:
This block shoudln't be indented under the `versionadded` notation.

Reply:
Fixed.

Reply:
Edit: now the CI fails https://github.com/django/django/actions/runs/18998302848/job/54261301068?pr=20033 and https://app.readthedocs.org/projects/django/builds/30162590/